openslides_backend/action/actions/user/base_json_upload.py

```diff
@@ -138,18 +143,22 @@ def validate_entry(self, entry: dict[str, Any]) -> dict[str, Any]:
 
                 self.row_state = ImportState.DONE
                 entry["id"] = id_
-                entry["username"] = {
-                    "value": username,
-                    "info": ImportState.DONE,
-                    "id": id_,
-                }
+                if isinstance(username, str):
```

Comment:
IMO we should restructure the validate-method. It's to long and complex.

Reply:
I agree with you, however, given that I will likely not have the required leisure to do a proper code refactoring on the user imports in the near future and that this issue is somewhat high in priority, I would suggest opening a cleanup issue for that.

Reply:
That's okay for me